exporter/opentelemetry-exporter-datadog/src/opentelemetry/exporter/datadog/version.py

```diff
@@ -12,4 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__ = "0.29b0"
+__version__ = "0.30b0"
```

Comment:
This package is deprecated, right? We don't want to release this anymore?

Reply:
Good point. I will undo this change and also create a task to exclude it from the release script.

Reply:
https://github.com/open-telemetry/opentelemetry-python-contrib/issues/1058